### PR TITLE
Initial component set up for consuming c3 charts as Angular2 component.

### PR DESCRIPTION
**Pull request by arunkumars08**
Features utility methods and cofiguration for handling line charts

@sudharsanam

### CODE REVIEW

File: ng2c3.component.ts.ts [hunk 1]
```diff
@@ -0,0 +1,80 @@
+import {Component, Input, SimpleChange, ElementRef, ViewEncapsulation} from '@angular/core';
+import * as c3 from 'c3';
```
Comment (ghost) [inline ng2c3.component.ts.ts:2]: Do we need this now? Or we have to figure out a way to write it for this widget

**Comment by ghost** [comment]
format seems to be ts.ts?

File: ng2c3.component.ts.ts [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+  @Input()
+  private data : any; // Configuration for series to be used for generating C3 has to be here
+  private element : HTMLElement; // Element to which the chart has to be attached to
+
+  constructor(elementReference : ElementRef) {
+    this.element = elementReference.nativeElement;
+  }
+
+  private __render( inputData : Map<string, any>) : void {
```
Comment (ghost) [inline ng2c3.component.ts.ts:31]: Can you modify the format of inputData to any? Same has been used at top for data

File: ng2c3.component.ts.ts (hunk 1 above)
Comment (arunkumars08) [reply ng2c3.component.ts.ts:2]: I can't quite understand which part you are referring to! Are you saying about the inclusion of 'c3'?

Comment (arunkumars08) [reply ng2c3.component.ts.ts:2]: Addressed that part.

File: ng2c3.component.ts.ts (hunk 2 above)
Comment (arunkumars08) [reply ng2c3.component.ts.ts:31]: Addressed

**Comment by ghost** [comment]
LGTM and 👍

**Comment by ghost** [comment]
Please merge this!

**Pull request merged**